src/main/java/com/salesforce/functions/jvm/sdk/data/bulk/BulkIngestTable.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.salesforce.functions.jvm.sdk.data.bulk;
+
+public interface BulkIngestTable {}
```

Comment:
`BulkIngestTable<T>`?

Reply:
also, will this contain:
* `Collection<String> getColumns()`
* `Collection<Collection<String>> getRows()`
or something else?